setup.py

```diff
@@ -87,9 +87,25 @@ def is_platform_windows():
     extra_link_args=extra_link_args,
 )
 
-utils_extension = Extension(
-    "pyuvdata._utils",
-    sources=["src/pyuvdata/utils.pyx"],
+bls_extension = Extension(
+    "pyuvdata._bls",
```

Comment:
another thing we can consider is to put this at `pyuvdata.utils._bls`.

It does technically  keep some of the compiled extensions out of the base namespace, a black belt user would not be able to do things like 
```python 
from pyuvdata import _coordinates
````
but it's a pretty minor point. Just food for thought.

Reply:
sounds good, I'll make that change.